cilium/README.md

```diff
@@ -43,6 +42,28 @@ To collect the cluster-wide `cilium-operator` metrics, see the [Cluster Agent][9
 
 2. [Restart the Agent][5].
 
+#### Log Collection
+Cilium contains two types of logs, `cilium-agent` logs and `cilium-operator` logs.
+
+#### Available for Agent >6.0
+1. Collecting logs is disabled by default in the Datadog Agent. Enable it in your [daemonset configuration][4]:
+
+    ```
+      (...)
+        env:
+          (...)
+          - name: DD_LOGS_ENABLED
+              value: "true"
+          - name: DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL
```

Comment:
Why do we need to enable this? Also using `DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL` will prevent setting a source and a service which can prevent correct parsing/usage of the logs

Reply:
This is the logs configuration to automatically collect logs from container environments. We document the same for istio, nginx-ingress-controller, etc.

Reply:
I think we need to document how to add annotations on the cilium pods directly. Otherwise we can't get the correct `source` attribute. Also people can selectively send logs to datadog instead of sending literally everything.